### ISSUE TITLE
ERROR in app: Exception on request POST /generate

### ISSUE THREAD

**Issue by folse** [e2b-dev/e2b#14]
When I try to run the agent, it show this error in terminal :

![image](https://user-images.githubusercontent.com/701198/230539817-16c0ae27-f734-4cb6-97de-ce0af7d912de.png)

**Screenshots**
![image](https://user-images.githubusercontent.com/701198/230539892-318168df-269c-4c4c-9223-af769409bc51.png)

Mac 13.2.1

**Comment by mlejva** [comment]
Hey, thanks for reporting this. This looks like it might be a bug on our backend. Is this happening every time you try to run it?

**Comment by mlejva** [comment]
Also @folse, can you please share your terminal output when you run `yarn dev` just to make sure every required service is running? Thank you

**Comment by folse** [comment]
> Hey, thanks for reporting this. This looks like it might be a bug on our backend. Specifically when data is being serialized before inserted into the database. Is this happening every time you try to run it?

Yes, every time.


![image](https://user-images.githubusercontent.com/701198/230556873-2c22168a-36b9-49ef-bc60-565c6acfac81.png)

![image](https://user-images.githubusercontent.com/701198/230557067-6e9244c5-98ac-4d02-9561-d1bcdaf7218f.png)

![image](https://user-images.githubusercontent.com/701198/230556936-3157c4b7-8936-4c9f-b9f4-3b4e7f17e189.png)

**Comment by mlejva** [comment]
Thanks! Since it's happening every time, it looks like one of the required services isn't running.

Based on the this:
![Screen Shot 2023-04-07 at 8 52 20 AM](https://user-images.githubusercontent.com/5136688/230557591-dcc65bd9-47e6-4296-9926-a8507b08f238.png)
it seems that your `.env` file isn't correctly formatted. Can you check that?

Two more questions:
1. When you followed the [get started guide](https://github.com/e2b-dev/e2b/blob/master/README.md#get-started) was there any warning or error?
2. Can you share the output of your browser developer tools console after you get the error?

**Comment by folse** [comment]
1. When you followed the [get started guide](https://github.com/e2b-dev/e2b/blob/master/README.md#get-started) was there any warning or error?

![image](https://user-images.githubusercontent.com/701198/230562885-d0892039-185b-43b6-b8bf-5e1b683645ea.png)

But I solved it by 'pip install lxml==4.9.2' in poetry shell.
And it looks I have solved the 'Python-dotenv could not parse statement starting at line 10/13' .


![image](https://user-images.githubusercontent.com/701198/230563128-dead1f0f-ca46-4faa-ad59-0b17b199c3c7.png)
![image](https://user-images.githubusercontent.com/701198/230563197-d66d0e89-4d4b-44bd-8480-749cd38e16ab.png)



3. Can you share the output of your browser developer tools console after you get the error?

![image](https://user-images.githubusercontent.com/701198/230562331-e763fa2f-843e-4905-8317-0132789a888e.png)

**Comment by folse** [comment]
Maybe is the network issue?

![image](https://user-images.githubusercontent.com/701198/230564435-08a849de-6fa1-4cdb-9d31-052c0231bc6f.png)

**Comment by ValentaTomas** [comment]
Can you `git pull` on master and try starting the project with the [new instructions](https://github.com/e2b-dev/e2b#get-started)? 
We added a way to start the app via Docker to solve a lot of dependency and connection issues people had.

You essentially just need to do the two steps:

## 1. Add env var
Create .env file by copying the [.env.example](https://github.com/e2b-dev/e2b/blob/master/.env.example)
```
cp .env.example .env
```
and fill in the following variable:
- OPENAI_API_KEY - your [OpenAI key](https://platform.openai.com/account/api-keys)

## 2. Start the app
```
npm start
```
Then open the page on http://localhost:3000/.

**Comment by ValentaTomas** [comment]
Hey @folse, there is a new update on `master` - we are no proxying all requests so a lot of network issues should be now resolved.

**Comment by folse** [comment]
Thanks Tomas Valenta,

after git pull , and the two steps, I got 

![image](https://user-images.githubusercontent.com/701198/231331205-ae5bc3f7-d3b1-4da1-8890-dc15f3cbd285.png)

No need to run `npm run db:start` ?

**Comment by mlejva** [comment]
Can you please do a clean `git clone` and try it again @folse? It looks like there are some build artifacts that are causing issues

**Comment by ValentaTomas** [comment]
> Can you please do a clean git clone and try it again @folse? It looks like there are some build artifacts that are causing issues

Yes, I think a clean git clone on `main` should help here. In the picture you provided it seems like the `npm start` is actually starting `next start` but the [actual script on `main` should do something else](https://github.com/e2b-dev/e2b/blob/5e5d8eb1bbaf4eb801c6de4e7130ca92052a0bfc/package.json#L15).

**Comment by mlejva** [comment]
Hey @folse, I'm closing this due to inactivity but please feel free to ask for this to get re-opened, post a new issue or ask on our [Discord](https://discord.gg/U7KEcGErtQ)
